lib/asciidoctor/converter/html5.rb

```diff
@@ -237,6 +211,41 @@ def document node
         end
         result << '</div>'
       end
+      
+      # Load Javascript at the end of body for performance
+      # See http://www.html5rocks.com/en/tutorials/speed/script-loading/
+      case node.attr 'source-highlighter'
+      when 'highlightjs', 'highlight.js'
+        highlightjs_path = node.attr 'highlightjsdir', %(#{cdn_base}/highlight.js/8.4)
+        result << %(<script src="#{highlightjs_path}/highlight.min.js"></script>
+<script>hljs.initHighlightingOnLoad()</script>)
```

Comment:
Now that the script is at the bottom of the page, we can change this method call to `hljs.initHighlighting()`. No need for the "OnLoad" part as we are already loaded :)